source/modules/MEI.xml

```diff
@@ -3112,16 +3112,16 @@
       </rng:data>
     </content>
   </macroSpec>
-  <macroSpec ident="data.OTHERSTAFF" module="MEI" type="dt">
-    <desc>For musical material designated to appear on another staff, the location of the staff
-      relative to the current one; <abbr>i.e.</abbr>, the staff above or the staff below.</desc>
+  <macroSpec ident="data.NEIGHBORINGLAYER" module="MEI" type="dt">
+    <desc>For musical material designated to appear on an adjacent layer or staff, the location of the layer
```

Comment:
Perhaps "an adjacent layer on a different staff"?

Reply:
this would always require to cross the staff, which isn't always necessary. Therefore: no ;-)